README.md

```diff
@@ -29,6 +27,12 @@ The target pane that text is sent to follows the standard `send-keys` format, so
 ```
 :lua require('slimux').configure_target_pane('0.1')
 ```
+
+Additionally, some effort is taken to automatically smart-escape input to send it to its destination intact. The array of escaped strings can be overridden in the above setup function, with the default setting being the below:
```

Comment:
is there anything more to the smartness than the default settings?

Reply:
Unfortunately no, not at the moment! So maybe smart-escape is overselling it a bit 😅. It's currently just iterating over that array and running a `gsub` to match against each string and prefix it with `\`.